source/fundamentals/time-series.txt

```diff
@@ -0,0 +1,134 @@
+
+Time series collections efficiently store sequences of measurements over
+a period of time. The collection consists of time series data with the
+following information:
+
+- Data collected over time
+- Metadata that describes the measurement
+- Date of the measurement
+
+.. list-table::
+   :widths: 33, 33, 33
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Example
+     - Measurement
+     - Metadata
+
+   * - Sales Data
+     - Revenue
+     - Company
+
+   * - Infection Rates
+     - Amount of People Infected
+     - Location
+
+Create a Time Series Collection
+-------------------------------
+
+To create a time series collection, pass the following parameters to the
+``CreateCollection()`` function:
+
+- The name of the new collection to create
+- The ``TimeSeriesOptions`` object specifying at least the time field
+
+.. important::
+
+   Versions prior to MongoDB 5.0 cannot create a time series collection.
+
+Example
+~~~~~~~
+
+The following example creates the ``spring_weather.march2022`` time series
+collection the ``temperature`` as the time field:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/timeSeries.go
+   :start-after: begin create ts coll
+   :end-before: end create ts coll
+   :emphasize-lines: 2, 5
+   :language: go
+   :dedent:
+
+To check if you created the collection, send the ``"listCollections"``
+command to the ``RunCommand()`` function:
+
+.. literalinclude:: /includes/fundamentals/code-snippets/timeSeries.go
+   :start-after: begin check ts coll
+   :end-before: end check ts coll
+   :emphasize-lines: 1, 4
+   :language: go
+   :dedent:
+
+After running this example, the output resembles the following:
+
+.. code-block:: json
+   :emphasize-lines: 9, 11, 15
+
+   {  
+      ...
+      "cursor": {
+           "firstBatch": [
+               {
+                   "info": {
+                       "readOnly": false
+                   },
+                   "name": "march2022",
+                   "options": {
+                       "timeseries": {
+                           ...
+                       }
+                   },
+                   "type": "timeseries"
+               },
+       ...
+    }
+
+Query a Time Series Collection
+------------------------------
+
+To query in a time series collection, use the same conventions as you
+would for :ref:`retrieving <retrieve-golang>` and aggregating data.
+
+Additional Information
+----------------------
+
+For more information on the operations mentioned, see the following
+guides:
+
+- :ref:`manual-timeseries-collection`
+- :ref:`manual-timeseries-collection-limitations`
+- :ref:`run-command-usage-example`
+- :ref:`retrieve-golang`
+
+.. TODO: db and collections, run command fundamentals page, aggregation page
+
+API Documentation
+~~~~~~~~~~~~~~~~~
+
+For more information on any of the functions or types discussed in this
+guide, see the following API Documentation:
+
+- `Database() <{+api+}/mongo#Database.Client>`__
```

Comment:
issue: I don't personally find this relevant to the concept. Probably safe to remove.